packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -190,7 +194,18 @@ function renderElement({attributes, children, element}: RenderElementProps) {
       return <tr {...attributes}>{children}</tr>
 
     case BlockFormat.TableCell:
-      return <td {...attributes}>{children}</td>
+      // TODO custom borderColor using colorPicker
+      return (
+        <td
+          {...attributes}
+          style={{
+            borderColor: (c => (c === 'transparent' ? `rgb(0, 0, 0, 0.1)` : c))(
```

Comment:
What does `c` stand for? Maybe try using more clear variable-names since it makes reading your code easier :)